### PR TITLE
added __ifloordiv__

### LINKED ISSUE #10308: __ifloordiv__

### PR DESCRIPTION
**Pull request by rarisenpai**
closes #10308

### CODE REVIEW

**Comment by rarisenpai** [comment]
Are there any changes required

File: ivy/functional/frontends/numpy/ndarray/ndarray.py [hunk 1]
```diff
@@ -388,6 +388,9 @@ def __imul__(self, value, /):
     def __itruediv__(self, value, /):
         return np_frontend.true_divide(self._ivy_array, value)
 
+    def __ifloordiv__(self, value, /):
+        return np_frontend.floor_divide(self.ivy_array, value, out=self)
```
Comment (zaeemansari70) [inline ivy/functional/frontends/numpy/ndarray/ndarray.py:392]: This should be `self._ivy_array`  in floor_divide(), and also remove the `out=self`.

Thanks!

**Review by zaeemansari70 — CHANGES_REQUESTED** [review]
Requested some changes.

**Review by zaeemansari70 — APPROVED** [review]
floor_div is not failing on the CI
lgtm!

**Pull request merged**